tests/resources/configs/manufactured-finetune_prithvi_swin_B_segmentation.yaml

```diff
@@ -99,8 +99,10 @@ model:
       num_frames: 1
       num_classes: 2
       head_dropout: 0.5708022831486758
-        #output_on_inference: probabilities #both
-    output_most_probable: False
+    output_on_inference: 
```

Comment:
@Joao-L-S-Almeida we need - if we add functionality to the config file - always update

    - documentation
    - json schema